docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,313 @@
+# Troubleshooting guide
+
```

Comment:
```suggestion

- [Troubleshooting guide](#troubleshooting-guide)
  * [Enable debug logs](#enable-debug-logs)
    + [core](#core)
    + [msgid](#msgid)
    + [socketclient](#socketclient)
    + [msgtable](#msgtable)
    + [statsclient](#statsclient)
    + [proxy](#proxy)
    + [Binapi generator debug logs](#binapi-generator-debug-logs)
  * [Connection problems](#connection-problems)
    + [Socket file does not exist](#socket-file-does-not-exist)
    + [Connection refused](#connection-refused)
    + [Permission denied](#permission-denied)
    + [The socket is not ready](#the-socket-is-not-ready)
  * [Problems with sending/receiving messages](#problems-with-sending-receiving-messages)
    + [Unknown message error](#unknown-message-error)
    + [VPP not replying to some messages](#vpp-not-replying-to-some-messages)
  * [Message versioning, deprecated and in-progesss messages](#message-versioning--deprecated-and-in-progesss-messages)
    + [Message versioning](#message-versioning)
    + [Deprecated messages](#deprecated-messages)
    + [In-progress messages](#in-progress-messages)

```

Reply:
Agreed, +1 for the TOC
Other than that lgtm

Reply:
TOC added